bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -201,14 +202,13 @@ private func dummy() {
     _ = NSLocalizedString("Navigate Up", comment: "")
     _ = NSLocalizedString("Navigates to custom color picker", comment: "")
     _ = NSLocalizedString("Navigates to customize the gradient", comment: "")
-    _ = NSLocalizedString("Navigates to select %s", comment: "translators: %s: Select control button label e.g. \"Button width\"")
+    _ = NSLocalizedString("Navigates to select %s", comment: "translators: %s: Select control button label e.g. Small\ntranslators: %s: Select control button label e.g. \"Button width\"")
     _ = NSLocalizedString("Navigates to the previous content sheet", comment: "")
     _ = NSLocalizedString("No application can handle this request.", comment: "")
     _ = NSLocalizedString("No application can handle this request. Please install a Web browser.", comment: "")
     _ = NSLocalizedString("No blocks found", comment: "")
     _ = NSLocalizedString("No custom placeholder set", comment: "")
     _ = NSLocalizedString("No preview available", comment: "")
-    _ = NSLocalizedString("None", comment: "")
```

Comment:
- Removal
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-alignment-control/ui.js#L34:L34)

This string is already translated as it's used in the web version, so we don't need to include it here.